typescript/infra/scripts/announce-validators.ts

```diff
@@ -118,7 +117,7 @@ async function main() {
     const location = announcement.value.storage_location;
     const announcedLocations =
       await validatorAnnounce.getAnnouncedStorageLocations([address]);
-    assert(announcedLocations.length == 1);
+    console.assert(announcedLocations.length == 1);
```

Comment:
Thanks for the fix but this is still not quite what it should be. The problem was the import. It should be `import {assert} from '@hyperlane-xyz/utils`

Reply:
Tks. I've fixed it!